src/05-invoicing/invoices.apib

```diff
@@ -215,17 +220,18 @@ Update an invoice.
 
     + Attributes (object)
         + id: `b7023c11-455e-4fa5-bb96-87f37dbc7d07` (string, required)
-        + customer (object, required) - Who is the invoice for?
-            + type (enum[string])
-                + Members
-                    + contact
-                    + company
-            + id: `2cc40a9c-e674-43db-9b15-447ccb478200` (string)
+        + invoicee (object, required)
+            + customer (object, required)
+                + type (enum[string])
+                    + Members
+                        + contact
+                        + company
+                + id: `ebafa4c5-fa8a-4409-92e5-1b192243372f` (string)
+            + for_attention_of (object, required)
```

Comment:
Shouldn't this be optional?

Reply:
So if you create an invoice without a `for_attention_of`, you can't update it without adding one?

I guess this should be or optional or nullable?

Reply:
Right now I'm thinking `nullable` ... but then I think we should do the same in the create and make it required but nullable?